packages/app/src/AppNav.tsx

```diff
@@ -808,41 +851,54 @@ export default function AppNav({ fixed = false }: { fixed?: boolean }) {
                         query.config !=
                           JSON.stringify(REDIS_DASHBOARD_CONFIG) &&
                         query.config != JSON.stringify(MONGO_DASHBOARD_CONFIG)
-                        ? 'text-success fw-bold'
-                        : 'text-muted-hover',
+                        ? [styles.listLinkActive]
+                        : null,
                     )}
                   >
-                    <i className="bi bi-plus me-2" />
-                    New Dashboard
+                    <div className="my-1">
+                      <i className="bi bi-plus me-2" />
+                      New Dashboard
+                    </div>
                   </a>
                 </Link>
-                <div className="fw-bold text-light fs-8 ms-3 mt-3">
-                  SAVED DASHBOARDS
-                </div>
-                {(dashboards ?? []).length === 0 ? (
-                  <div className="text-muted ms-3 mt-2">0 saved dashboards</div>
+                <Input
+                  placeholder="Saved Dashboards"
+                  variant="unstyled"
+                  size="xs"
+                  ml={4}
+                  icon={<i className="bi bi-search fs-8 ps-1" />}
+                  rightSection={
+                    dashboardsListQ && (
+                      <CloseButton
+                        size="xs"
+                        onClick={() => setDashboardsListQ('')}
+                      />
+                    )
+                  }
+                  value={dashboardsListQ}
+                  onChange={e => setDashboardsListQ(e.currentTarget.value)}
+                />
+                {dashboards.length === 0 ? (
+                  <div className="text-slate-300 fs-8 p-2 px-3">
+                    No saved dashboards
```

Comment:
Nit: Should probably match the `No results matching <i>{searchesListQ}</i>` style above